rel/overlay/etc/vm.args

```diff
@@ -38,8 +38,7 @@
 {{node_name}}
 
 # All nodes must share the same magic cookie for distributed Erlang to work.
-# Uncomment the following line and append a securely generated random value.
-# -setcookie
+{{erlang_cookie}}
```

Comment:
Hi,

can this change break "couchdb-pkg" somewhere, if we are maybe grepping for "-setcookie"? @rnewson

Reply:
I have not worked with `couchdb-pkg` before but I can verify this concern there.

Reply:
https://github.com/apache/couchdb-pkg/blob/d284420d445573c726b4fcf1d8940ce22793e364/rpm/SPECS/couchdb.spec.in#L155

https://github.com/apache/couchdb-pkg/blob/d284420d445573c726b4fcf1d8940ce22793e364/debian/couchdb.postinst#L165

https://github.com/apache/couchdb-pkg/blob/d284420d445573c726b4fcf1d8940ce22793e364/debian/couchdb.config#L85

Reply:
Thanks @jan for the references.

Reply:
The package scripts should be (_need to be_)_ robust against this kind of thing and I believe they are.

If the commented-out setcookie line is no longer present after this change the packaging scripts will append a setcookie line to the end of the file. That's less tidy than it being where we planned for it to be, but it will still work.

I would, however, prefer for this change not to have unintended side-effects at all.